02-peer-protocol.md

```diff
@@ -174,6 +174,7 @@ The sending node:
   - MUST set `push_msat` to equal or less than 1000 * `funding_satoshis`.
   - MUST set `funding_pubkey`, `revocation_basepoint`, `htlc_basepoint`, `payment_basepoint`, and `delayed_payment_basepoint` to valid DER-encoded, compressed, secp256k1 pubkeys.
   - MUST set `first_per_commitment_point` to the per-commitment point to be used for the initial commitment transaction, derived as specified in [BOLT #3](03-transactions.md#per-commitment-secret-requirements).
+  - MUST set `channel_reserve_satoshis` greater than or equal to `dust_limit_satoshis`.
```

Comment:
typo: "MUST set `channel_reserve_satoshis` [to]  greater than"?

Reply:
Then we get a repeated "to" (to greater than or equal to).  We can omit the final to but that reads worse to me...